src/test/resources/application-test.yml

```diff
@@ -8,7 +8,7 @@ management.endpoint:
 web-client:
   configuration:
     max-retries: 2
-    back-off: PT1S
+    back-off: PT0.25S
```

Comment:
Makes the tests run quicker.

Reply:
is that 10 and a quarter seconds?

Reply:
No its 0.25 of a sec.

Reply:
Looks like just a quarter second to me?

Reply:
For context the code retries failed requests after applying a back off. I set this to 1s for the test context originally but there are quite a few tests using this now it it slows down the test run quite a bit so I decreased it.